requirements.txt

```diff
@@ -8,6 +8,7 @@ cherrypy
 portend
 chardet
 gntp
+pygobject
```

Comment:
This needs to be conditional, on Windows this can't be installed.

Reply:
Good catch, I forget about windoze all the time :grin:

Also added a minimum version to ensure proper threading support, [3.10.2](https://ftp.acc.umu.se/pub/GNOME/sources/pygobject/3.10/pygobject-3.10.2.changes) is from 2013 so that shouldn't be a problem.